django/db/migrations/utils.py

```diff
@@ -0,0 +1,28 @@
+from django.apps import apps
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.loader import MigrationLoader
+from django.db.migrations.state import ProjectState
+
+
+def check_unmigrated_models(loader=None, connection=None):
+    """
+    :return: A (possibly empty) list of messages,
+    if the currently registered models have changes that are not
+    yet represented in a migration
+    """
+    messages = []
+    if loader is None:
+        loader = MigrationLoader(connection)
+
+    autodetector = MigrationAutodetector(
+        loader.project_state(),
+        ProjectState.from_apps(apps),
+    )
+    if autodetector.changes(graph=loader.graph):
+        messages.extend([
```

Comment:
@kezabelle Thank you for the question. The presence of the comma at the end of line 24 means there are two strings / messages in the list, hence the use of `.extend()` instead of `.append()`. 

I will agree that a trailing comma at the end of a string causing that string to be wrapped in a tuple (or collection/list type here) can be one of the less explicitly obvious features of Python. I did consider in development putting an extra newline character in there to make the two messages more explicitly obvious, so if there are additional changes to get this into Django I'll throw it in (but I won't update the 3 different patches for such a minor coding style/readability issue unless additional work is actually needed on this patch).

This is present to preserve backwards compatibility with the previous migrations code as much as possible, as otherwise I would need to join them into a single message somehow, and as Django supports Windows where IIRC you print `\r\n`, or carriage-return + line feed to the console instead of just `\n` or line feed in OSX/Linux/Unix environments, I think it's the simplest way forward to just keep printing two separate messages. 

For reference please see the existing migrations code on master at this time, this code should be tested as being functionally equivalent in what it prints to the console but please point out if I need to add any further relevant test coverage:
https://github.com/django/django/blob/4dcfbd792342667174834c40e792f1b8ac2f0609/django/core/management/commands/migrate.py#L183-L191